.lintstagedrc.js

```diff
@@ -0,0 +1,16 @@
+// lint-staged.config.js
+const { ESLint } = require('eslint');
+
+const removeIgnoredFiles = async (files) => {
+  const eslint = new ESLint();
+  const ignoredFiles = await Promise.all(files.map((file) => eslint.isPathIgnored(file)));
+  const filteredFiles = files.filter((_, i) => !ignoredFiles[i]);
+  return filteredFiles.join(' ');
+};
+
```

Comment:
I believe that by adding this line in tsconfig it is not necessary to change eslint default configuration. It adds seed.ts and won't throw any warning. 
![image](https://github.com/sirius-valley/tricker-backend/assets/104390122/1dcce785-f188-45a9-9f6f-1690e94dc997)